scripts/common.sh

```
@@ -0,0 +1,483 @@
+    else
+        echo "No valid RAID configuration found"
+        rm -f "/etc/mdadm/mdadm.conf.new"
+    fi
+
+    return 0
+}
+
+get_kernel_version() {
+    local deb_file="$1"
+    local version=""
+    local basename_file=$(basename "$deb_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)